content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -17,15 +17,26 @@ aliases:
 
 ## Lassie
 
-Lassie is a simple retrieval client for Filecoin. It finds and fetches your data over the best retrieval protocols available.
+Lassie is a simple retrieval client for IPFS and Filecoin. It finds and fetches your data over the best retrieval protocols available. Lassie makes Filecoin retrieval easy. While Lassie is powerful, the core functionality is expressed in a single CLI command:
 
-### Prerequisites
+```shell
+lassie fetch <CID>
+```
+
+Lassie fetches content in content-addressed archive (CAR) form, so in most cases you will need additional tooling to deal with CAR files.
```

Comment:
a bit of an odd statement right after "Lassie makes Filecoin retrieval easy"... oy maybe we need to just add car extraction directly.